tools/kotlin-native-gradle-plugin/build.gradle

```diff
@@ -166,5 +166,9 @@ gradlePlugin {
             id = 'konan'
             implementationClass = 'org.jetbrains.kotlin.gradle.plugin.KonanPlugin'
         }
+        create("kotlin-native") {
+            id = 'kotlin-native'
```

Comment:
Gradle recommends to add a "package like" name.
Like you do with other plugins (https://plugins.gradle.org/plugin/org.jetbrains.kotlin.jvm or https://plugins.gradle.org/plugin/org.jetbrains.dokka).
Should we use `org.jetbrains.kotlin-native` here? 🤔

Reply:
"Big" Kotlin provides both namings for its plugins ([repo](https://github.com/JetBrains/kotlin/tree/master/libraries/tools/kotlin-gradle-plugin/src/main/resources/META-INF/gradle-plugins)) but only plugins with package-like names are published to the Gradle plugin portal. I think we can use the same approach here + add the portal publication for our plugin.

Reply:
Don't know if we just follow the "big brother" instead of doing it "right"/"correct" from the beginning?!